custom_components/bluetti_bt/translations/en.json

```diff
@@ -3,7 +3,9 @@
     "step": {
       "init": {
         "data": {
-          "use_controls": "Use controls (use at own risk)"
+          "use_controls": "Use controls (use at own risk)",
+          "persistant_conn": "Persistant connection",
```

Comment:
Minor, but it's spelt 'persistent' (without the 'a')

Reply:
fixed